src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
I'm sorry I've tried to replace "in" for "of" but its not working correctly((